docs/pages/dsCompositeIn/ds-compositeIn-L3-Low-Level-TestSpec.md

```diff
@@ -100,6 +101,6 @@ classDiagram
   - This configuration file contains the list of requirements for tests to execute. Eg: Copying the streams, setting environment variables etc.
```

Comment:
could you please check whether this file handles copying the streams as mentioned in example , if not please remove the copying files from example.